Dockerfile.stable

```diff
@@ -1,82 +1,86 @@
 FROM cfmeqe/sel_base_fc29
 
-#TODO: figure why user 0 was needed
-USER 0
+ENV SELENIUM_MAJOR_VERSION=3 \
+    SELENIUM_MINOR_VERSION=141 \
```

Comment:
Note - in ManageIQ/integration_tests browser use, it looks like there are some changes in this selenium upgrade that cause an exception on is_displayed use in Chrome.

I was able to work around this by setting ChromeOption kwarg `w3c` to `False`